ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/sno/provision/cucushift-installer-rehearse-ibmcloud-ipi-sno-provision-chain.yaml

```diff
@@ -0,0 +1,16 @@
+chain:
+  as: cucushift-installer-rehearse-ibmcloud-ipi-sno-provision
+  steps:
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: ipi-conf-ibmcloud
+  - ref: ipi-conf-ibmcloud-manual-creds  
+  - ref: single-node-conf-ibmcloud
```

Comment:
can we use https://steps.ci.openshift.org/reference/ipi-conf-ibmcloud to set the master and worker replicas directly? just like what did for other platforms